content/pages/office-policies.md

```diff
@@ -0,0 +1,50 @@
+Title: Office Policies
+Category: pages
+Status: hidden
+Slug: office policies
+
+
+WiCS OFFICE POLICY (Version 2.0 March 8th 2015) 
+
+PERSONNEL:
+
+* If the office is open for office hours, drop-ins or for any other reason, a MINIMUM of ONE committee member of WiCS must be present in the office at all times. 
```

Comment:
Please try to justify lines to 80 character width. If you can provide more information about which text editor you're using, I can help you set it up to do this automatically.